### ISSUE TITLE
[Guide UI] Should checkboxes for terms and clickable terms be swapped?

### ISSUE THREAD

**Issue by MeemawHustlin** [MichaelEbert/OblivionProgressTracker#203]
Checkboxes are smaller and harder to quickly and accurately check off compared to clicking on the overall terms. I feel like our best option might be to make the checkbox be to the left of the term and have both toggle the checking off, and have a separate little ? icon to the right of the term that brings up the extra info, since in most cases we will need to be quickly clicking on several terms to check them off.

**Comment by MichaelEbert** [comment]
GENIUS

**Comment by MichaelEbert** [comment]
Is the checklist good as-is? or should we also use the '❔' icon there?

**Comment by MeemawHustlin** [comment]
Yeah the actual checklist page is fine, we don't really need extra context on that page. The casual guide would theoretically have all that they need from there, and the checklist would mostly function as an overview of their progress at that point.

**Comment by MichaelEbert** [comment]
This also solves the map link problem: map link can just be an additional icon

**Comment by MichaelEbert** [comment]
'🗺️' for map?

**Comment by MeemawHustlin** [comment]
🗺️ looks better to me.

**Comment by MeemawHustlin** [comment]
The popout question mark and the clickable full term have both been implemented, so I'm going to close this
